.github/workflows/eventarc-audit-storage.yaml

```diff
@@ -0,0 +1,45 @@
+name: eventarc-audit-storage
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request:
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request_target:
+    types: [labeled]
+  schedule:
+  - cron:  '0 2 * * *'
+jobs:
+  test:
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'actions:force-run' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - uses: 'google-github-actions/auth@v0.3.1'
```

Comment:
We recommend pinning actions to `@vx` (`@v0` in this case) unless you have security concerns, in which case you should pin to a specific SHA. This applies to all workflows and google-github-actions, but I'll only leave this comment once.